x-pack/test/fleet_api_integration/apis/index.js

```diff
@@ -17,20 +17,8 @@ export default function ({ loadTestFile, getService }) {
     // Fleet setup
     loadTestFile(require.resolve('./fleet_setup')); // ~ 6s
 
-    // Enrollment API keys
-    loadTestFile(require.resolve('./enrollment_api_keys/crud'));
-
-    // Package policies
-    loadTestFile(require.resolve('./policy_secrets'));
-    loadTestFile(require.resolve('./package_policy/create'));
-    loadTestFile(require.resolve('./package_policy/update'));
-    loadTestFile(require.resolve('./package_policy/get'));
-    loadTestFile(require.resolve('./package_policy/delete'));
-    loadTestFile(require.resolve('./package_policy/upgrade'));
-    loadTestFile(require.resolve('./package_policy/input_package_create_upgrade'));
-
-    // Agent policies
-    loadTestFile(require.resolve('./agent_policy'));
```

Comment:
are these rows removed by mistake?

Reply:
No, they were merged by mistake in the last PR, the structure of the fleet test configs has changed e.g agent_policy is now called [here](https://github.com/hop-dev/kibana/blob/1c777a2e702f7c5ae9c1b86063515c0a27368efb/x-pack/test/fleet_api_integration/config.agent_policy.ts#L15), enrollment_api_keys_crud by 
[ the line below](https://github.com/hop-dev/kibana/blob/96a698afb5a5aace11f9bbd51359ee66068428aa/x-pack/test/fleet_api_integration/apis/index.js#L22)  and all the package policy tests [here](https://github.com/hop-dev/kibana/blob/1c777a2e702f7c5ae9c1b86063515c0a27368efb/x-pack/test/fleet_api_integration/config.package_policy.ts#L15)